### ISSUE TITLE
XWindow support

### ISSUE THREAD

**Issue by MasonProtter** [blinksh/blink#569]
Currently no ios SSH apps support XWindow forwarding so you can't view graphical objects like graphs or what have you over the SSH connection. It'd be *really* nice this app were to offer such a feature.

**Comment by yury** [comment]
@andrius I use bpcopy/bppaste setup with tmux and it works between iPad/tmux/vim.

will look on xclip, but it should work.

**Comment by andrius** [comment]
Could you please share some of your configs? Actually it does not work for me and that make me sad :) (only works properly within tmux/vim (and between tmux/vim/mac) (yeah, i also use pbcopy at mac and vim with X support at vps)

**Comment by andrius** [comment]
> I use bpcopy/bppaste setup with tmux and it works between iPad/tmux/vim.

@yury how do you copy from vps/tmux back to the ipad? (there is no issues pasting to the vps from ipad)

**Comment by yury** [comment]
I’m on the way back to home. Will send my configs as soon as I get to iPad ;)

It works only with ssh. Release version of mosh doesn’t support it yet (but code is already there).

**Comment by yury** [comment]
Hmm, it stops working for me... will investigate with debugger tomorrow. I think I broke smth with my setup.

**Comment by andrius** [comment]
@yury Maybe it would be good idea to create wiki with description for all the featueres, i.e. external display or clipboard exchange. I think many of us would be happy to collaborate and fill up it. In my case I am very curious regarding both external screen (as I see it is fixed as for now, last year I were opened issue) and with clipboard (vim/tmux -> ipad) 

Regarding clipboard, there is online guides how to make it working between vim and tmux and between vim and mac/linux (using xorg or using tmux). It works both on mosh and ssh session, but I never get success with blink and ipad

**Comment by yury** [comment]
@andrius yes, we are thinking about improving documentation. After 12 release it is hight priority for us. There are so many features blink app has, but no one knows about :)

about copying... It is strange it stops working.
quick experiments showed that this perl script works https://github.com/skaji/remote-pbcopy-iterm2.
But I don't like that it replaced original command. Will look more on cross platform solution.
